app/controllers/chargeback_controller.rb

```diff
@@ -197,6 +197,7 @@ def cb_rate_edit
           detail.metric = r[:metric]
           detail.chargeback_rate_detail_measure_id = r[:chargeback_rate_detail_measure_id]
           detail.chargeback_rate_detail_currency_id = r[:chargeback_rate_detail_currency_id]
+          @sb[:rate_details][:currency] = r.detail_currency.code
```

Comment:
We want to avoid querying like this in a loop by preloading the associations we'll be calling like I show above.